docs/site/clojure-lsp.md

```diff
@@ -4,15 +4,15 @@ Calva uses a mix of static and dynamic analysis to power the experience. A lot o
 
 ## Starting the LSP server
 
-You don't need to do anything to start clojure-lsp. No install, no commands, no nothing. It does take a while for clojure-lsp to start, though, especially the first time for a new project, when clojure-lsp (via `clj-kondo`) indexes the project files.
+You don't need to do anything to start clojure-lsp. No install, no commands, no nothing. Calva will download the correct binary for your operating system if necessary (this should only happen when the clojure-lsp version is updated in a new release of Calva), and then start it. It does take a while for clojure-lsp to start, though, especially the first time for a new project, when clojure-lsp (via `clj-kondo`) indexes the project files.
```

Comment:
```suggestion
You don't need to do anything to start clojure-lsp. No install, no commands, no nothing. Calva will download the correct binary for your operating system if necessary (this should only happen when the clojure-lsp version is updated in a new release of Calva), and then starts it. It does take a while for clojure-lsp to start, though, especially the first time for a new project, when clojure-lsp (via `clj-kondo`) indexes the project files.
```

Reply:
Reworded a bit to say "Calva downloads... then starts." I think it did make sense grammatically before as "Calva will download, then [will] start," but now it's maybe a bit more clear.